src/index.ts

```diff
@@ -118,6 +118,7 @@ async function createPullRequest(
     gitHubConfigs.primary
   );
   logger.info('Finished PR workflow');
```

Comment:
Unrelated to this PR: make this logging message consistent with above?
```suggestion
  logger.info(
    `Successfully opened pull request available at url: ${pullResponseData.url}.`
  );
```